README.md

```diff
@@ -214,6 +200,9 @@ For all commits with a tag that follows the pattern `\d+\.\d+\.\d+` the job cont
 
   * Working but not deeply tested base version 
 
+## Origins
+
+The workflow is a port of the Roddy-based [https://github.com/TheRoddyWMS/BamToFastqPlugin](BamToFastqPlugin). Compared to the Roddy-workflow, some problems with the execution of parallel processing, resulting in potential errors, have been fixed.
```

Comment:
convention of usage of round and square brackets seems to be borken. could you please double check this?

Reply:
Right. It's not a convention, but Markdown syntax. So this is actually an error. Thanks for spotting this!